src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj

```diff
@@ -34,24 +35,14 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <Compile Include="System\Diagnostics\StackTraceSymbols.CoreCLR.cs" />
-    <!--
-      The following references are only facades that align references for the PCL
-      libraries (Immutable/MetadataReader) this library is referencing. We reference
-      the depproj's with targetgroup netcore50 because we need these to be the older
-      version that matches what we shipped for netcore50. We cannot reference the live
-      csproj's because the version numbers are higher and aren't all supported on netcore50.
-    -->
-    <ProjectReference Include="..\..\System.Collections\src\redist\System.Collections.depproj">
-      <TargetGroup>netcore50</TargetGroup>
-    </ProjectReference>
-    <ProjectReference Include="..\..\System.IO\src\redist\System.IO.depproj">
-      <TargetGroup>netcore50</TargetGroup>
-    </ProjectReference>
-    <ProjectReference Include="..\..\System.Runtime\src\redist\System.Runtime.depproj">
-      <TargetGroup>netcore50</TargetGroup>
+    <ProjectReference Include="..\..\System.Collections\src\System.Collections.csproj" />
```

Comment:
The netcore50 build configuration did not exist in the .builds file, even before my change. Is there anything that needs fixing?